### PR TITLE
Show patch

### PR DESCRIPTION
**Pull request by garrettwrong**
Looks like I missed the `show()` command in the image volume return PR.

This seems to fix it up and adds tests so it shouldn't happen again.

### CODE REVIEW

**Comment by garrettwrong** [comment]
Confirmed this fixes the issue I encountered (and was later reported by CI).

File: tests/test_image.py [hunk 1]
```diff
@@ -281,3 +283,14 @@ def testMultiDimBroadcast():
...
     X = mdim_ims + ims
     assert np.allclose(X[0], 2 * ims.asnumpy())
+
+
+def testShow():
+    """
+    Test show doesn't crash.
+    """
+    # Use non GUI backend.
+    matplotlib.use("Agg")
```
Comment (janden) [inline tests/test_image.py]: Stuff into a decorator? May need to do this more if we have new display functionality.

**Review by janden — CHANGES_REQUESTED** [review]
Looks good. Just one suggestion.

File: tests/test_image.py (hunk 1 above)
Comment (garrettwrong) [reply tests/test_image.py]: (cracks knuckles) my pleasure

Comment (janden) [reply tests/test_image.py]: Nice!

**Review by janden — APPROVED** [review]

**Review by j-c-c — APPROVED** [review]
LGTM!

**Pull request merged**